core/services/chainlink/application.go

```diff
@@ -39,9 +39,12 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting/types"
 	"go.uber.org/multierr"
+	"go.uber.org/zap/zapcore"
 	"gopkg.in/guregu/null.v4"
 )
 
+var serviceLoggers = map[string]interface{}{}
```

Comment:
why the global? Seems like we just want this as global application state i.e. living on the ChainlinkApplication object

Reply:
removing, wasn't necessary